prox_config_backup.sh

```diff
@@ -2,7 +2,6 @@
 # Version	      0.2.2 - BETA ! !
 # Date		      02.20.2020
 # Author 	      DerDanilo 
-# Contributors    bootsie123
```

Comment:
You can push this again, I am fine with contributions :)

Reply:
Alrighty then. I also added to the list the previous users who submitted pull requests